components/GlobalStyle/GlobalStyle.js

```diff
@@ -0,0 +1,18 @@
+import { createGlobalStyle } from "styled-components";
+
+export const GlobalStyle = createGlobalStyle`
+  * {
+      box-sizing: border-box;
+      --darkblue: #2d2861;
+      --orange: #fe7902;
+      --lightyellow: #fcc13f;
+      --green: #fe7902;
+      --lightgreen: #8ad234;
```

Comment:
I guess this should be in root? 😊

Reply:
Thank you for your suggestions Jelena :)